addons/mrp_subcontracting_purchase/tests/test_mrp_subcontracting_purchase.py

```diff
@@ -644,3 +646,59 @@ def test_update_qty_purchased_with_subcontracted_product(self):
         self.assertEqual(picking.backorder_ids.state, 'cancel')
         po.order_line.product_qty = 2.0
         self.assertEqual(po.order_line.product_qty, 2.0)
+
+    @freeze_time('2024-01-01')
+    def test_bom_overview_availability(self):
```

Comment:
Could you give another name than the one of you test in mrp_subcontracting ?

Reply:
Good idea, I've updated the name :+1: